tools/webpack.config.ts

```diff
@@ -16,9 +17,13 @@ function getHotCSS(bundle: any[], isProd: boolean): any[] {
     ].concat(bundle);
 }
 
-export default (env?: string): webpack.Configuration => {
+interface Configuration extends webpack.Configuration {
+    devServer?: webpackDevServer.Configuration;
```

Comment:
Does webpack types really does not have types for `devServer` config? I'd expect them to have that since it part of webpack configuration.

Reply:
There are typescript types avalible for webpack-dev-server at `@types/webpack-dev-server`. I think you can use them instead of having to import `webpack-dev-server` since that's and extra dependency that we don't want to install in production.

Reply:
The devServer member was moved out of webpack into webpack-dev-server in one of the new versions.